msteams-platform/apps-in-teams-meetings/build-apps-for-anonymous-user.md

```diff
@@ -0,0 +1,222 @@
+
+> [!NOTE]
+>
+> * When an anonymous user is added to the meeting, `membersAdded` payload object doesn't have `aadObjectId` field.
+> * When an anonymous user is added to the meeting, `from` object in the payload always have the id of the meeting organizer, even if the anonymous user was added by another presenter.
+
+**ConversationUpdate Event – MembersRemoved Payload object**
+
+```csharp
+{
+  "membersRemoved": [
+    {
+      "id": "<<GUID1>>" 
+    }
+  ],
+  "type": "conversationUpdate",
+  "timestamp": "<<timestamp>>",
+  "id": "<<event unique identifier>>",
+  "channelId": "msteams",
+  "serviceUrl": "<<serviceURL>>",
+  "from": {
+    "id": "<<GUID2>>"
+  },
+  "conversation": {
+    "isGroup": true,
+    "tenantId": "<<tenant id>>",
+    "id": "<<conversation id>>"
+  },
+  "recipient": {
+    "id": "<<bot id>>",
+    "name": "<<bot name>>"
+  },
+  "channelData": {
+    "tenant": {
+      "id": "<<tenant id>>"
+    },
+    "source": null,
+    "meeting": {
+      "id": "<<meeting id>>"
+    }
+  }
+}
+```
+
+| **Property Name** | **Description** |
+| --- | --- |
+| `id` under `membersRemoved` | Anonymous user ID |
+| `id` under `from` | Meeting organizer ID |
+| `tenantId` | Tenant ID of the meeting organizer |
+| `id` under `conversation` | Conversation ID |
+| `id` under `recipient` | Bot ID |
+| `id` under `tenant` | Tenant ID of the meeting organizer |
+
+> [!NOTE]
+>
+> * When an anonymous user is removed from a meeting, `membersRemoved` payload object doesn't have `aadObjectId` field.
+> * When an anonymous user is removed from a meeting, `from` object in the payload always have the id of the meeting organizer, even if the anonymous user was removed by another presenter.
+
+**Create Conversation API**
+
+Create conversation API is not supported for anonymous users. If a bot attempts to create a conversation with an anonymous user, it will receive a `400` Bad Request status code, with the following payload:
+
+```csharp
+{ 
+  "error": {
+    "code": "BadArgument",
+    "message": "Bot cannot create a conversation with an anonymous user"
+  }
+} 
+```
+
+**Invoke**
+
+Anonymous users can interact with Adaptive Cards, which use invoke activities. When an invoke request is sent from anonymous user to the bot, the payload request will not have `imdisplayname`.
+
+For more information on [invoke activities with Adaptive card](/microsoftteams/platform/task-modules-and-cards/cards/cards-actions?tabs=json).
+
+## Adaptive card for anonymous users
+
+Currently, anonymous users can't see the app icon when the message is sent through a bot app. It displays only generic app icon as following:
+
+:::image type="content" source="../assets/images/apps-in-meetings/app-icon.png" alt-text="This screenshot shows you how the app icon displays for anonymous user.":::
+
+## See also
+
+* [Build apps for Teams meeting stage](build-apps-for-teams-meeting-stage.md)
+* [Meeting apps APIs](meeting-apps-apis.md)
```

Comment:
add a link for [How Microsoft Teams bots work](https://learn.microsoft.com/en-us/azure/bot-service/bot-builder-basics-teams?view=azure-bot-service-4.0&tabs=csharp)